### PR TITLE
Fix phy update implementation and test cases

### PR DESCRIPTION
**Pull request by bihanssen**
Fixed typo in phy_update implementation that lead to failing non-expected update values. The tests now run and pass as expected.

### CODE REVIEW

**Comment by bihanssen** [comment]
Resolves #159.

File: tests/test_phy_update.py [hunk 1]
```diff
@@ -47,56 +47,51 @@
...
+    def start(self, connect_with, req_phys):
         self.connect_with = connect_with
         logger.info(f"scan_start, trying to find {self.connect_with}")
         scan_duration = 5
         self.adapter.driver.ble_gap_scan_start()
         try:
             self.conn_handle = self.conn_q.get(timeout=scan_duration)
-            resp = self.adapter.phy_update(self.conn_handle, req_phy)
-            self.new_phy = resp 
+            logging.info(f"Central requesting: {req_phys=}")
```
Comment (chunfantasy) [inline tests/test_phy_update.py:81]: What does the equal sign mean?

Comment (bihanssen) [reply tests/test_phy_update.py:81]: The equal sign is a relatively new feature in Python that prints the expression as well as the value.

**Review by chunfantasy — APPROVED** [review]
LGTM

**Review by chunfantasy — APPROVED** [review]

**Pull request merged**